src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockDefault2.java

```diff
@@ -43,21 +43,21 @@ void method2(int a) {
     void method3(int a, int b) {
         switch (a) {
             case 1: break;
-            default: {} method2(a);     // violation
+            default: {} method2(a);     // violation 'Must have at least one statement'
         }
 
         switch (b) {
             case 2: break;
             default: method2(b); {}     // ok
         }
 
-        switch (a+b) {case 1: break; default: {} ; }    // violation
+        switch (a+b) {case 1: break; default: {} ; }    // violation 'Must have at least one statement'
```

Comment:
remove extra spaces before `// violation`

Reply:
@romani  Done . Similary can you tell me how to shorten it in InputEmptyBlockSemantic2Statement file.

Reply:
`System.identityHashCode("a")` replace with any other statement like `a++;`

Reply:
Anything that is not about blocks can be changed.